docs/atlascli/command/atlas-organizations-invitations-invite.txt

```diff
@@ -65,7 +65,7 @@ Options
    * - --role
      - strings
      - true
-     - User's roles  for the associated organization. To learn which values the Atlas CLI accepts, see the Items Enum for roles in the API spec: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#operation/inviteOneMongoDBUserToJoinOneAtlasOrganization/.
+     - User's roles  for the associated organization. To learn which values the Atlas CLI accepts, see the Items Enum for roles in the API spec: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#operation/createOrganizationInvitation/.
```

Comment:
This doc link opens only the spec without pointing to the correct operation. Am I missing something?
The correct link seems this one to me [inviteOneMongoDBUserToJoinOneAtlasOrganization](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Organizations/operation/inviteOneMongoDBUserToJoinOneAtlasOrganization)

Reply:
Hi @andreaangiolillo The operationId changes will go live today and the links will only after today's release. Thanks!